cmd/files/cmd_files.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/Smartling/smartling-cli/services/files"
 	"github.com/Smartling/smartling-cli/services/helpers/rlog"
 
+	sdkjobs "github.com/Smartling/api-sdk-go/api/jobs"
```

Comment:
The SDK name is incorrect. We have two APIs: Jobs and Job Batches. Please be explicit in naming. You must use Job Batches SDK in this file.